rasa/train.py

```diff
@@ -158,18 +159,19 @@ def train(
         )
 
     old_model = model.get_latest_model(output_path)
-    retrain_core, retrain_nlu = model.should_retrain(
+    retrain_core, retrain_nlu, retrain_nlg = model.should_retrain(
         new_fingerprint, old_model, train_path
     )
 
-    if force_training or retrain_core or retrain_nlu:
+    if force_training or retrain_core or retrain_nlu or retrain_nlg:
```

Comment:
optional: I find this a tiny bit easier / faster to read:
```suggestion
    if any([force_training, retrain_core, retrain_nlu, retrain_nlg]):
```